platform-db/src/main/java/com/softicar/platform/db/runtime/table/logic/DbTableRowDeleter.java

```diff
@@ -84,6 +84,7 @@ private void deleteRows() {
 
 			// send post-delete notification
 			sendNotification(IDbTableListener::afterDelete, rows);
+			DbTableRowCommitNotifier.addNotification(table, DbTableRowNotificationType.CHANGE, rows);
```

Comment:
The `[I]DbTableRowNotificationSet` can now (obviously) contain several differently-typed notifications for any given row. I wonder if we should warn about that in the Javadoc of `IDbTableRowNotificationSet.[getDeletedRows|getChangedRows]` because calling code might need to implement deduplication.

Reply:
added Javadoc to `IDbTableRowNotificationSet`

Reply:
okay